test_runner/src/main/kotlin/ftl/args/IArgs.kt

```diff
@@ -40,7 +40,11 @@ interface IArgs {
     val useLegacyJUnitResult: Boolean get() = false
     val ignoreFailedTests: Boolean
     val keepFilePath: Boolean
-    val outputStyle: OutputStyle? get() = null
+    val outputStyle: OutputStyle
+    val defaultOutputStyle
+        get() = if (flakyTestAttempts > 0 || (!disableSharding && maxTestShards > 0))
```

Comment:
what about additional test apks? If we have additional test apks, then that'll involve multiple matrices.